<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIScalePrefs</class>
 <widget class="PrefWidget" name="UIScalePrefs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>307</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>32767</width>
    <height>32767</height>
   </size>
  </property>
  <property name="whatsThis">
   <string>Threshold for raising edge.&lt;p&gt;
You can enter your values with a suffix like m, u, n, p, k, M, G, T&lt;br&gt;Example:&lt;br&gt;
10k - 10000&lt;br&gt;100m - 0.1</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <item>
    <widget class="QGroupBox" name="ButtonGroup36">
     <property name="title">
      <string>Scale</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
      <item>
       <widget class="QRadioButton" name="autoScaleBut">
        <property name="whatsThis">
         <string>Select this entry if you want automatic scaling of the vertical scale in the transient recorder.</string>
        </property>
        <property name="text">
         <string>Au&amp;tomatic</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="TextLabel9_2">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ui_includeZero">
          <property name="text">
           <string>Include zero line</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="Spacer14">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="manualScaleBut">
        <property name="whatsThis">
         <string>Select this entry if you want to set the minimum and maximum value for the scale manually. If selected enter the minimum and maximum values below.</string>
        </property>
        <property name="text">
         <string>&amp;Manual</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="TextLabel9">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="TextLabel10">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>M&amp;inimum:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>ui_scaleMin</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ui_scaleMin">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>32767</height>
           </size>
          </property>
          <property name="whatsThis">
           <string>Enter the minimum scale value here.&lt;p&gt;
You can enter your values with a suffix like m, u, n, p, k, M, G, T&lt;br&gt;Example:&lt;br&gt;
10k - 10000&lt;br&gt;100m - 0.1</string>
          </property>
          <property name="text">
           <string>-3.999</string>
          </property>
          <property name="maxLength">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="TextLabel10_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Ma&amp;ximum:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>ui_scaleMax</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ui_scaleMax">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>32767</height>
           </size>
          </property>
          <property name="whatsThis">
           <string>Enter the maximum scale value here.&lt;p&gt;
You can enter your values with a suffix like m, u, n, p, k, M, G, T&lt;br&gt;Example:&lt;br&gt;
10k - 10000&lt;br&gt;100m - 0.1</string>
          </property>
          <property name="text">
           <string>3.999</string>
          </property>
          <property name="maxLength">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="Spacer7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ButtonGroup37">
     <property name="title">
      <string>Window</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>10</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="TextLabel11">
        <property name="text">
         <string>&amp;Size:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>ui_winSize</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="TextLabel12">
        <property name="text">
         <string>Max. &amp;length:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>winLength</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="Spacer8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="sizeUnit">
        <property name="whatsThis">
         <string>Unit for visible window width.</string>
        </property>
        <item>
         <property name="text">
          <string>Seconds</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Minutes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Days</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="lengthUnit">
        <item>
         <property name="text">
          <string>Seconds</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Minutes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Days</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="ui_winSize">
        <property name="whatsThis">
         <string>Size (in time) of the visible part of the recorder graph. See unit to the right.</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
        <property name="value">
         <number>600</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="winLength">
        <property name="whatsThis">
         <string>Maximum length (in time) of the recorder graph. If this time is reached the first recorded values will be discarded. It will not stop recording.</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
        <property name="value">
         <number>3600</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="Spacer5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PrefWidget</class>
   <extends>QWidget</extends>
   <header>prefwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>autoScaleBut</tabstop>
  <tabstop>manualScaleBut</tabstop>
  <tabstop>ui_scaleMin</tabstop>
  <tabstop>ui_scaleMax</tabstop>
  <tabstop>ui_winSize</tabstop>
  <tabstop>sizeUnit</tabstop>
  <tabstop>winLength</tabstop>
  <tabstop>lengthUnit</tabstop>
 </tabstops>
 <includes>
  <include location="global">prefwidget.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>manualScaleBut</sender>
   <signal>toggled(bool)</signal>
   <receiver>TextLabel10</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualScaleBut</sender>
   <signal>toggled(bool)</signal>
   <receiver>ui_scaleMin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualScaleBut</sender>
   <signal>toggled(bool)</signal>
   <receiver>TextLabel10_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualScaleBut</sender>
   <signal>toggled(bool)</signal>
   <receiver>ui_scaleMax</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoScaleBut</sender>
   <signal>toggled(bool)</signal>
   <receiver>ui_includeZero</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
